src/components/wizard-navigation-bar.component.html

```diff
@@ -10,7 +10,8 @@
         done: isDone(step),
         editing: isEditing(step),
         optional: isOptional(step),
-        navigable: isNavigable(step)
+        navigable: isNavigable(step),
+        completion: isCompletionStep(step)
```

Comment:
You've added a new css class `completion` to the `li` html elements.
What is missing in my opinion are stylesheets making use of the new css class in https://github.com/madoar/angular-archwizard/blob/master/src/components/wizard-navigation-bar.component.horizontal.less and https://github.com/madoar/angular-archwizard/blob/master/src/components/wizard-navigation-bar.component.vertical.less

Reply:
It's a leftover from trials. It's useless, I propose to remove it, especially if it causes issues.